Makefile.am

```diff
@@ -1,6 +1,6 @@
 ACLOCAL_AMFLAGS = -I m4
 AM_CPPFLAGS = -I$(top_srcdir)/libass
-AM_CFLAGS = -std=gnu99 -Wall -Wextra -Wno-sign-compare -Wno-unused-parameter \
+AM_CFLAGS = -Wall -Wextra -Wno-sign-compare -Wno-unused-parameter \
```

Comment:
Is there any benefit to this other than removing 11 bytes from `Makefile.am`?
I consider explicitly specifying the required C standard good practice and do not like depending on current defaults of external tools.

Reply:
Explicitly requesting C99 means we won't get C11, which may be needed for atomics.

Reply:
Is there any platform where selecting `GNU99` or `C99` disables `stdatomic.h`?
Furthermore, autoconf only enables GNU11 or C11 by default since 2.70, so if that’s really needed an explicit check for GNU11/C11 with allowing a fallback to GNU99/C99 would not only be more explicit about what we want/need but also more backwards-compatible.